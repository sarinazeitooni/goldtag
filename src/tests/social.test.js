import React from 'react';
import {shallow} from 'enzyme';
import Social from "../components/social/social";
describe("social component" , ()=>{
    it("should match snap shot" , ()=>{
        const wrapper = shallow(<Social/>);
        expect(wrapper).toMatchSnapshot();
    });
    it("should render correctly" , ()=>{
        const wrapper = shallow(<Social/>);
        const container = wrapper.find('div.social-container');
        expect(container.exists()).toBe(true);
    });
    it("should render image correctly" , ()=>{
       const wrapper = shallow(<Social/>);
       const image = wrapper.find("div.social-icon");
       expect(image.exists()).toBe(true);
    });
})