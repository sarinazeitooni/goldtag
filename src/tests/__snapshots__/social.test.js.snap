// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`social component should match snap shot 1`] = `ShallowWrapper {}`;
